src/components/App/index.js

```diff
@@ -163,25 +164,12 @@ export class App extends Component {
 			return language;
 		};
 
-		const browserLanguage = () => (navigator.userLanguage || navigator.language);
-
-		const normalizeLanguageString = (languageString) => {
-			let [languageCode, countryCode] = languageString.split ? languageString.split(/[-_]/) : [];
-			if (!languageCode || languageCode.length !== 2) {
-				return 'en';
-			}
-			languageCode = languageCode.toLowerCase();
-
-			if (!countryCode || countryCode.length !== 2) {
-				countryCode = null;
-			} else {
-				countryCode = countryCode.toUpperCase();
-			}
-
-			return countryCode ? `${ languageCode }_${ countryCode }` : languageCode;
+		const iframeLanguage = () => {
```

Comment:
Since you're calling `updateWidgetLanguage()`, I think you don't need this block of code here, right?